### PR TITLE
feat: [PLATO-621] sign up banner

### PR DESCRIPTION
**Pull request by stephanLeece**
## Purpose of PR

As a user browsing the Starter Templates from the Marketing website, I want to have the ability to sign-up and auto-install a Starter Template right from its preview page, So that I can start using it as soon as possible with minimal efforts.

Therefore, we are adding a sticky banner, from which the user can visit the sign up page.

Ticket: https://contentful.atlassian.net/browse/PLATO-621

Desktop:
Screenshot 2023-05-09 at 14 44 28

Tablet:
Screenshot 2023-05-09 at 14 44 44

Mobile:
Screenshot 2023-05-09 at 14 44 55

## Testing / QA
Tested on https://template-marketing-webapp-nextjs-git-private-branchplato-399dfb.colorfuldemo.com/

- UI should match [designs](https://www.figma.com/file/pXmQhBNuCiovgEGkFwKuh3/Templates---Horizontal?type=design&node-id=10175-196124&t=gxIFwkFuJEIZjcEe-0) at each breakpoint
- Clicking the CTA should lead to sign up form link as defined [here](https://contentful.atlassian.net/browse/PLATO-617)
- Tracking event should fire on CTA click https://app.segment.com/contentful-staging/sources/segment_starter_templates/debugger

### CODE REVIEW

**Comment by roosterhack** [comment]
just spoke with Luigi, the CTA button should be the same as the one on the marketing website, including hover state, padding and font styles, colours etc. I added this requirement to the ticket.

File: src/_ctf-private/ctf_sign-up-banner/CtfSignUpBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+import clsx from 'clsx';
+
+import Logo from './CtfLogo.svg';
+
+import { Link } from '@src/components/shared/link';
+
+const signUpLink =
+  'https://www.contentful.com/starter-templates/marketing-website/sign-up/?action=create_starter_template&template_name=marketing';
+const systemUIFontFamilies =
+  '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"';
```
Comment (cf-remylenoir) [inline src/_ctf-private/ctf_sign-up-banner/CtfSignUpBanner.tsx:12]: I reckon from the design & the marketing website, the font family is

`"Avenir Next W01", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"`

**Comment by stephanLeece** [comment]
Cheers folks, i will just add the forma36 package so everything's spot on

**Comment by stephanLeece** [comment]
i'll steal the button and font styles from the marketing website. cheers!

**Review by roosterhack — APPROVED** [review]

**Review by cf-remylenoir — APPROVED** [review]
QA'ed with success ✨

**Pull request merged**